be/src/formats/parquet/column_reader.cpp

```diff
@@ -651,6 +978,7 @@ class StructColumnReader : public ColumnReader {
     // First non-nullptr child ColumnReader, used to get def & rep levels
     const std::unique_ptr<ColumnReader>* _def_rep_level_child_reader = nullptr;
     const ColumnReaderOptions& _opts;
+    std::map<std::string, size_t> _field_name_to_reader_idx;
```

Comment:
how about make `_child_reader` as a `map<string, ColumnReader>`

Reply:
The old code use index to get reader/subfiledcolumn/subfieldtype, I will use subfield name decouple them.